src/virt/emulator.rs

```diff
@@ -526,9 +531,6 @@ impl VirtContext {
         // The WFI instruction put the processor in a special state that enables taking interrupts
         // even if mstatus.MIE = 0. We keep a bit in the virtual context to model that state.
         self.is_wfi = true;
-        // After a WFI the execution always resumes on the next instruction, whether an interrupt
-        // is taken or not.
-        self.pc += 4;
```

Comment:
Is the moving of the `pc + 4` just refactoring or required for correctness (i.e. did it caused the verification?) @francois141